lib/galaxy/tools/parameters/basic.py

```diff
@@ -1271,7 +1271,7 @@ def get_options(self, trans, other_values):
             try:
                 with open(dataset.get_file_name(), 'r') as f:
                     head = f.readline()
-                cnames = head.rstrip().split('\t')
+                cnames = head.rstrip("\n\r ").split('\t')
```

Comment:
I would assume this as the default. The documentation says "whitespaces", is \r not included?

Reply:
Default is that `\n\r\t `. The problem is that this removes the first or last columns if they are empty.